hugo/content/docs/learn/getting-started.md

```diff
@@ -0,0 +1,180 @@
+---
+title: 'Getting Started'
+---
+
+**Welcome to the first step of your Sprotty journey!** In this section, we'll set up everything you need to start building interactive diagrams.
+
+![Project Setup](/images/project-setup.png)
+
+## Choose Your Adventure
+
+There are two ways to begin your Sprotty project:
+
+{{< tabs "setup-method" >}}
+{{< tab "Fast Track (Recommended)" >}}
+
+## 🚀 Fast Track with Yeoman
+
+If you want to jump straight into learning Sprotty concepts without manual setup, our Yeoman generator has you covered!
+
+### 1. Install the Generator
+
+First, install Yeoman and the Sprotty generator:
+
+```bash
+npm install -g yo generator-sprotty
+```
+
+### 2. Generate Your Project
+
+Run the generator and follow the prompts:
+
+```bash
+yo sprotty
+```
+
+The generator will ask you a few questions about your project. For this tutorial, you can accept the default options by pressing Enter.
+
+### 3. Explore Your New Project
+
+That's it! Your Sprotty project is now ready to go. The generator has created all the necessary files and installed all dependencies.
+
+**What's next?** Continue to the [Data Model]({{< ref "/docs/learn/data-model" >}}) section to understand the components of your generated project.
+
+{{< /tab >}}
+{{< tab "Step-by-Step" >}}
+
+## 🛠️ Step-by-Step Setup
+
+If you prefer to understand every piece of your project from the ground up, follow these steps to set up your project manually.
+
+### 1. Create a New Package
+
+First, create a new directory for your project and initialize a new npm package:
+
+```bash
+mkdir hello-world
+cd hello-world
+npm init -y
+```
+
+This will generate a `package.json` file at the root of your project. You can simplify it by replacing its content with:
+
+```json
+{
+  "name": "hello-world",
+  "version": "1.0.0",
+  "description": "A simple Sprotty diagram",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1"
+  }
+}
+```
+
+### 2. Set Up TypeScript
+
+Since we're using TypeScript, initialize the configuration:
+
+```bash
+npx tsc --init
+```
+
+### 3. Create Project Structure
+
+Create the following directory structure:
+
+```bash
+hello-world/
+├── src/         # TypeScript source files
+└── static/      # HTML, CSS, and compiled JS
+```
+
+```bash
+mkdir src static
+```
+
+### 4. Install Dependencies
+
+Install the required dependencies:
+
+```bash
+# Core dependencies
+npm install sprotty inversify reflect-metadata
```

Comment:
inversify comes already with sprotty. It could lead to problems when another version is added here.